logs_rollbar.admin.inc

```diff
@@ -52,6 +52,14 @@ function logs_rollbar_admin_settings($form, &$form_state) {
     '#default_value' => variable_get('logs_rollbar_auto_disable_on_local', TRUE),
   );
 
+  $form['logs_rollbar_ignored_messages'] = array(
+    '#type' => 'textfield',
+    '#size' => 2048,
+    '#title' => t('Ignored Messages'),
+    '#description' => t('Comma separated exceptions messages which Rollbar ignore'),
```

Comment:
We should note that it's for client-side.
https://github.com/rollbar/rollbar.js/issues/216
Also it's a regex in the end.

`Comma separated client-side exceptions messages which Rollbar ignores, regular expressions are supported'